packages/documentation/src/stories/components/buttons/close-button/close-button.docs.mdx

```diff
@@ -0,0 +1,34 @@
+import { Canvas, Controls, Meta } from '@storybook/blocks';
+import * as CloseButtonStories from './close-button.stories';
+
+<Meta of={CloseButtonStories} />
+
+<div className="docs-title">
+  # Close button
+
+  <nav>
+    <link-design of={JSON.stringify(CloseButtonStories)}></link-design>
+  </nav>
+</div>
+
+<div className="lead">
+  The close button is a web component. It is a slightly adapted version of the icon button, intended
+  for a specific use case.
+</div>
+
+<Canvas sourceState="shown" of={CloseButtonStories.Default} />
+<div className="hide-col-default">
+  <Controls of={CloseButtonStories.Default} />
+</div>
+
+## Usage in other components
```

Comment:
Now that I see it described on the docs page, the whole topic doesn't really seem to make life any easier for developers. Because they have to do everything as intended, but can only omit the definition of the slot prop on the `<post-closebutton>` component during implementation.

On the other hand, it can be a source of confusion because they need to know about this custom behaviour when reading the HTML code of a component using the close button.

My suggestion: Show the docs page to the team (e.g. in the daily), so they can decide if we really want to implement it or discard the idea.